irods/session.py

```diff
@@ -360,7 +360,7 @@ def connection_timeout(self, seconds):
         elif seconds is None:
             pass
         else:
-            exc = ValueError("The iRODS connection_timeout must be assigned to a positive int or float with units of seconds, or to None which effectively disables the timeout.")
+            exc = ValueError("The iRODS connection_timeout must be assigned a positive int or float, or None.")
```

Comment:
```suggestion
            exc = ValueError("The iRODS connection_timeout must be assigned a positive int, positive float, or None.")
```

Reply:
In the time since this review comment (which I didn't notice until too late) I've changed the text to:
`"The iRODS connection_timeout must be assigned a positive int or float value, or None."`
But I think I prefer the suggested change, so will change it to that.

Reply:
Done.